### ISSUE TITLE
Support "Remote Wipe" feature

### ISSUE THREAD

**Issue by simonspa** [daper/nextcloud-passwords-app#58]
With Nextcloud 17 they introduced a feature called "remote wipe" which allows you to remove all data from a client which identifies itself via an app password.

It would be nice (and quite sensible for a passwords app) to implement this.

**Comment by simonspa** [comment]
For reference: https://nextcloud.com/blog/nextcloud-17-brings-remote-wipe-collaborative-text-editor-and-next-generation-secure-watermarking/

**Comment by szaimen** [comment]
this feature will not work with single sign on https://github.com/daper/nextcloud-passwords-app/issues/63

**Comment by simonspa** [comment]
@szaimen why not? The official files app uses SSO and supports the feature - or am I missing something?

**Comment by szaimen** [comment]
> @szaimen why not? The official files app uses SSO and supports the feature - or am I missing something?

Yes, the official app has its own app token, which you need to use remote wipe. Currently the SSO library doesn't use nested app tokens to bring that feature also to other apps. So all apps that currently use the SSO library don't support remote wipe. 
This is server-side needed to get this functionality to the SSO library: https://github.com/nextcloud/server/issues/17459

So the current state is that you either support SSO or remote wipe.

**Comment by simonspa** [comment]
Ah, darn, thank you for the explanation. So the current way these two are implemented on the server side is a bit half-baked... :/
